src/utils/selectors.ts

```diff
@@ -155,9 +168,7 @@ export const experienceSelector = (state: RootState) =>
   state.firestore.data.experience;
 
 export const venueSelector = (state: RootState) =>
```

Comment:
currentVenueSelector?

Reply:
Should be changed in another PR there might be a lot of usage for this one.